src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/bulkscan/OcrValidationControllerTest.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.controllers.bulkscan;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Resources;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import uk.gov.hmcts.reform.authorisation.exceptions.InvalidTokenException;
+import uk.gov.hmcts.reform.finrem.caseorchestration.controllers.BaseControllerTest;
+import uk.gov.hmcts.reform.finrem.caseorchestration.error.ForbiddenException;
+import uk.gov.hmcts.reform.finrem.caseorchestration.error.UnauthenticatedException;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.AuthService;
+
+import java.io.IOException;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(OcrValidationController.class)
+public class OcrValidationControllerTest extends BaseControllerTest {
+
+    public static final String BULKSCAN_VALIDATION_BASIC_FORM_JSON = "fixtures/bulkscan/validation/basic-form.json";
+    public static final String BULkSCAN_VALIDATION_VALID_OCR_RESPONSE_JSON = "fixtures/bulkscan/validation/valid-ocr-response.json";
+
+    @MockBean
+    private AuthService authService;
+
+    @Before
+    public void setUp() {
+        super.setUp();
+    }
+
+    @Test
+    public void should_return_401_status_when_auth_service_throws_unauthenticated_exception() throws Exception {
+        String requestBody = readResource(BULKSCAN_VALIDATION_BASIC_FORM_JSON);
+        given(authService.authenticate("")).willThrow(UnauthenticatedException.class);
+
+        mvc
+                .perform(
```

Comment:
wow, 8 spaces?

Reply:
it was IntelliJ auto-formatting - maybe I'm missing some formatting setting, do you have formatting template for IntelliJ in divorce?

I'll fix it in the branch implementing validation rules.